Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "perf-zoo"
+version = "0.1.0"
+authors = ["Yao Yue <thinkingfish@gmail.com>"]
+edition = "2018"
+
+[dependencies]
```

Comment:
Consider defining separate binary targets, one per example:

```
[[bin]]
name = "mutex-contention"
path = "src/main.rs"
```

probably want to rename the main.rs to mutex-contention.rs or similar.